vsintegration/ProjectTemplates/TutorialProject/Template/Tutorial.fsx

```diff
@@ -156,9 +213,12 @@ module ListsAndArrays =
     /// all the days of the year.
     let daysList = 
         [ for month in 1 .. 12 do
-              for day in 1 .. System.DateTime.DaysInMonth(2012, month) do 
+              for day in 1 .. System.DateTime.DaysInMonth(2017, month) do 
                   yield System.DateTime(2012, month, day) ]
 
+    // Print the first 5 elements of 'daysList' using 'List.take'.
+    printfn "The first 5 days of 2017 are: %A" (daysList |> List.take 5)
```

Comment:
I was thinking about this, and it's kind of difficult to introduce piping (and composition with `>>`) without also having Lists/Arrays/etc, since those are some of the most natural data structures to process data via a pipeline.  I think that if we have pipelines prior to this section with some basic lists then it should be okay.